README.md

```diff
@@ -1,2 +1,233 @@
+       Authorization Endpoint - https://localhost:9443/oauth2/authorize
+       Token Endpoint - https://localhost:9443/oauth2/token
+       Userinfo Endpoint - https://localhost:9443/oauth2/userinfo
+       Logout Endpoint - https://localhost:9443/oidc/logout
+       Redirect URI - http://localhost:8080/callback/
+       PostLogout Redirect URI - http://localhost:8080/postlogout/
+5. Continue the on-screen guidance and complete the installation.
+
+### Running the sample
+Double click on the `Pickup Manager` application available on your Desktop.<br/>
+![pickup manager](https://user-images.githubusercontent.com/15249242/95334396-aa17c580-08cb-11eb-83ee-3b88b8512f68.gif)
+
+## How it works
+This section explains detailed walkthrough on how key aspects handled in the Asgardio .NET OIDC SDK.
+Througout this section we will refer to the source folder of the sample as <APP_HOME>
+
+  * [Trigger authentication](#trigger-authentication)
+  * [Retrieve user attributes](#retrieve-user-attributes)
+  * [Trigger logout](#trigger-logout)
+
+The structure of the sample would be as follows:<br/>
+![structure](https://user-images.githubusercontent.com/15249242/95333060-e34f3600-08c9-11eb-91e3-8ea64446eb77.PNG)
+
+### Trigger authentication
+In the `<APP_HOME>/LoginPage.xaml` page, we have registered a `Click` event named `LoginButton_Click` for the login button to trigger a OIDC authentication:
+```xml
+<Button x:Name ="login" Content="LOGIN" HorizontalAlignment="Left" Margin="73,452,0,0" VerticalAlignment="Top" 
+                Width="240" Height="56" FontFamily="Segoe UI Black" FontSize="22" Click="LoginButton_Click" Background="LightSeaGreen" Foreground="Black"/>
+```
+
+The button click would trigger an authentication request, and redirect the user to the IdP authentication page.
+Upon successful authentication, the user would be redirected to the application homepage.
+
+### Retrieve user attributes
+In the `<APP_HOME>/LoginPage.xaml.cs` file, we have added the following code inside the login button click trigger method to get the user subject value and the user attributes referring the SDK API.
+
+```csharp
+private async void LoginButton_Click(object sender, RoutedEventArgs e)
+{
+    // Redirect the user to IDP authentication page
+    await authenticationHelper.Login();
+
+    // Focus to app windows after succeful authentication
+    this.Activate();
+
+    // Retrieve access token and user information
+    accessToken = authenticationHelper.AccessToken;
+    userInfo = authenticationHelper.UserInfo;
+
+    // Display the home page window
+    HomePage home = new HomePage(accessToken, userInfo);
+    home.Show();
+    this.Close();
+}
+```
+
+### Trigger logout
+In the `<APP_HOME>/HomePage.xaml` file, we have added the following button to trigger a SLO flow:
+```xml
+<Button x:Name="logoutButton" Width="Auto" Foreground="White" Background="Black" FontWeight="Bold" 
+        FontSize="14" Click="Logout_button_click" Margin="944,10,9.6,650.2">
```

Comment:
Better if we could use a style class as mentioned before.

Reply:
Created https://github.com/wso2/product-is/issues/9803 to track this improvement